app/handlers/signup_password.rb

```diff
@@ -20,7 +20,11 @@ def authorized?
   end
 
   def handle
-    outputs.user = User.create
+    outputs.user = User.build
+    if signup_state.trusted?
+      outputs.user.full_name = signup_state.trusted_data['name']
+    end
```

Comment:
I think we still need to send the user through the profile screen (so they can change their name from "Bob Jones" to "Professor Bob Jones".  If so, can we do the name there? Prepopulate the name field on that form?